README.md

```diff
@@ -46,10 +46,14 @@ OPTIONS:
                 L1 height to fork the superchain (bounds L2 time). `0` for latest
 
           --chains value                                                         ($SUPERSIM_CHAINS)
-                chains to fork in the superchain, mainnet options: [base, lyra, metal, mode, op, orderly, pgn, zora]
+                chains to fork in the superchain, mainnet options: [base, lyra, metal, mode, op,
+                orderly, pgn, superlumio, zora]. In order to replace the public rpc endpoint for
+                a chain, specify the ($SUPERSIM_RPC_URL_<CHAIN>) env variable. i.e SUPERSIM_RPC_URL_OP=http://optimism-mainnet.infura.io/v3/<API-KEY>
 
           --network value                     (default: "mainnet")               ($SUPERSIM_NETWORK)
-                superchain network. options: mainnet, sepolia, sepolia-dev-0
+                superchain network. options: mainnet, sepolia, sepolia-dev-0. In order to
+                replace the public rpc endpoint for the network, specify the
+                ($SUPERSIM_RPC_URL_<NETWORK>) env variable. i.e SUPERSIM_RPC_URL_MAINNET=http://mainnet.infura.io/v3/<API-KEY>
```

Comment:
what is the difference between the chain rpc url (`$SUPERSIM_RPC_URL_<CHAIN>`) and the network url (`$SUPERSIM_RPC_URL_<NETWORK>`)? shouldnt there just be one rpc url that combines both chain + network?

Reply:
the former is for the L2 chains. The latter is for the L1 chain

Reply:
oh ok, I was kind of confused by that because you can also have an l2 "network". do you think that could be confusing? should we update to using l1 chain here instead of network?